doc/man/sphinx-build.rst

```diff
@@ -4,14 +4,18 @@ sphinx-build
 Synopsis
 --------
 
-**sphinx-build** [*options*] <*sourcedir*> <*outputdir*> [*filenames* ...]
+| **sphinx-build** -M <*builder*> <*sourcedir*> <*outputdir*> [*options*]
+| **sphinx-build** [*options*] <*sourcedir*> <*outputdir*> [*filenames* ...]
 
 Description
 -----------
 
 :program:`sphinx-build` generates documentation from the files in
 ``<sourcedir>`` and places it in the ``<outputdir>``.
 
+Please note that there are 2 signatures for 2 usage scenarios. ``-M`` flag
+invokes a :ref:`make mode <make_mode>`, otherwise it works in a *build mode*.
```

Comment:
```suggestion
The available modes are the *build-mode* via :option:`sphinx-build -b`
and the *make-mode* via :option:`sphinx-build -M`.
```

Reply:
I would also suggest adding at least a mention of the fact that signatures for those mods are different. This fact is obvious for long time users while often gets completely missed by new users and leads to hours of wasted time. A couple extra words in exchange for hours saved is IMO a good trade off.

Reply:
You can add this note at the end of sentence, e.g., "Note that these modes have a different signature. The *make-mode* always requires the -M argument to comes first."